utils/server/websocket.go

```diff
@@ -18,45 +18,45 @@ var upgrader = websocket.Upgrader{
 	},
 }
 
-func (s *server) handleWebsocket() http.HandlerFunc {
+func (s *Server) handleWebsocket() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		c, err := upgrader.Upgrade(w, r, nil)
 		if err != nil {
 			log.Println("upgrade:", err)
 			return
 		}
 
-		client := client.CreateWebsocketClient(c)
-		defer s.realtime.RemoveClient(client.ClientID())
-		defer s.functions.UnregisterService(client.ClientID())
+		wsClient := client.CreateWebsocketClient(c)
```

Comment:
Lets just rename the client to `c`?

Reply:
Alright but I had to rename the already c upgrader to u